### PR TITLE
Removing "State" from view API info

### PR DESCRIPTION
**Pull request by abidlabs**
Previously, the `State` component would appear in the `/info` route, which would show the incorrect syntax for how to use the clients, since a value for State should *not* be passed in through the clients. Neither should it be returned from the endpoint. 

This fixes that. I also release a new version of the API Fetcher Space to make this change backwards compatible and add a test.

Edit: I also separated the client tests into "flaky" and "non-flaky" to make diagnosing CI easier  

Closes: #3897

### CODE REVIEW

**Review by freddyaboulton — COMMENTED** [review]
The change looks good to me @abidlabs !

But I think we don't need a new server? We can just install this change in the current server?

File: client/python/gradio_client/utils.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 CONFIG_URL = "/config"
 API_INFO_URL = "/info"
 RAW_API_INFO_URL = "/info?serialize=False"
-SPACE_FETCHER_URL = "https://gradio-space-api-fetcher-v2.hf.space/api"
+SPACE_FETCHER_URL = "https://gradio-space-api-fetcher-v2-1.hf.space/api"
```
[the remark this answers is not in the document]
Comment (abidlabs) [reply client/python/gradio_client/utils.py:32]: Yes that could work too. But I wanted to ensure that all the tests would pass

[the remark this answers is not in the document]
Comment (abidlabs) [reply client/python/gradio_client/utils.py:32]: And didn't want to interfere with the existing server while it was running

**Review by freddyaboulton — APPROVED** [review]
Thanks for the fix @abidlabs !

**Pull request merged**